multiqc/modules/base_module.py

```diff
@@ -162,7 +162,7 @@ def clean_s_name(self, s_name, root):
             root = ''
         # Split then take first section to remove everything after these matches
         for ext in config.fn_clean_exts:
-            s_name = s_name.split(ext ,1)[0]
+            s_name = os.path.basename(s_name.split(ext ,1)[0])
```

Comment:
I don't have any problems with this, but I'm surprised that it makes a difference.. The `s_name` usually comes from the `find_log_files` function which should supply the base filename. I've not seen directory paths without the `-d` option myself before, but if you were getting that then I guess this addition doesn't hurt.